code/modules/mob/living/carbon/human/species.dm

```diff
@@ -958,9 +959,17 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/check_species_weakness(obj/item, mob/living/attacker)
 	return 0 //This is not a boolean, it's the multiplier for the damage that the user takes from the item.It is added onto the check_weakness value of the mob, and then the force of the item is multiplied by this value
 
+/**
+ * Soft equip the outfit required for life. Soft equip will equip if the slot is free, or drop to floor.
+ */ 
+/datum/species/proc/give_important_for_life(mob/living/carbon/human/human_to_equip)
+	if(outfit_important_for_life)
+		outfit_important_for_life= new()
+		outfit_important_for_life.equip(human_to_equip)
```

Comment:
Like this maybe? That way we don't create a new datum every time this is called.
```
/datum/species/proc/give_important_for_life(mob/living/carbon/human/human_to_equip)
	if(!outfit_important_for_life)
		return
	if(ispath(outfit_important_for_life))
		outfit_important_for_life = new outfit_important_for_life()
	outfit_important_for_life.equip(human_to_equip)

Reply:
Creating new outfit datums every time is currently the norm and this method is not called very often, I don't see a problem with it as-is